src/commands/selector/replay.ts

```diff
@@ -0,0 +1,125 @@
+import {TokenAwareCommand} from '../../lib/token-command'
+import {initClient} from '../../lib/client'
+import { ux, Flags } from '@oclif/core'
+
+
+type ReplayEvent = {
+  entity: string
+  key: string
+  event: string
+  eventId: string
+  timestamp: string
+  meta: Record<string, string>
+  data: Record<string, any>
+}
+
+export default class SelectorReplay extends TokenAwareCommand {
+  static description = 'Replay an entities events.'
+
+  /* eslint-disable quotes */
+  static examples = [
+    `$ evently selector:replay \
+  -n article \
+  -e add-comment \
+  -k author`,
+    `$ evently selector:replay \
+  -n article \
+  -e add-comment -e delete-comment \
+  -k author, -k date \
+  --limit 10`,
+  ]
+
+  static flags = {
+    entity: Flags.string({
+      char:         'n',
+      description:  'Entity name',
+      required:     true,
+    }),
+    event: Flags .string({
+      char:         'e',
+      description:  'Event name',
+      required:     false,
+      multiple:     true,
+    }),
+    key: Flags.string({
+      char:         'k',
+      description:  'Entity keys to select for.',
+      required:     true,
+      multiple:     true,
+    }),
+    after: Flags.string({
+      description: 'Select events that occur after this ledger mark or event ID.',
+    }),
+    limit: Flags.integer({
```

Comment:
Add `char: 'l'` ?

Reply:
limit should have a default of some kind, perhaps 50. As well as a max? A selector could blow up RAM.

Reply:
Maybe a max should be enforced server-side? Happy to add it do the client through

Reply:
It can't be server-side, as clients will stream large numbers of events through reducer functions to produce state. However, in the CLI, you probably never want large numbers of events. You _may_ want to see the 'last 50' or similar, but I haven't built that yet.